tests/negative/regression/filesystem.go

```diff
@@ -31,7 +31,7 @@ func VFATIgnoresWipeFilesystem() types.Test {
 	out := in
 	mntDevices := []types.MntDevice{
 		{
-			Label:        "EFI-SYSTEM",
+			Label:        "OEM",
```

Comment:
Should we move this into a separate commit?

Reply:
Just split it into a new commit.